tests/test_typescript_features.py

```
@@ -0,0 +1,524 @@
+async function fetchData(url: string): Promise<string> {
+    const response = await fetch(url);
+    if (!response.ok) {
+        throw new Error("Failed");
+    }
+    const data = await response.text();
+    return data;
+}
+"""
+
+
+def run_tldr(args: list[str], cwd: str = None) -> dict:
```

Ruff (0.14.11)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)